### DISCUSSION TITLE
Uno Figma and how to use with a Windows FluentUI

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#14843] [General]
Hi, looking at the Uno Figma plugin, great idea.  My project targets Windows 10/11 Fluent UI and iPadOS applications.

We would like to use the Fluent UI standard for our application since the majority of users are Windows, but we feel that in the future more people will migrate to the iPad version.  But we have to develop to today.

Your base plugin is themed as Material, which is not what we want to use, how can we change that in the figma UI so our page layouts look like a fluent UI?

Next, we have several custom controls that we use in our application, how can we add those to the toolkit and have it generate the code for them, most of our controls are just wrappers on existing WinUI3 controls but have special properties such as permissions or are a combination of controls (Label above the text input for example).

Again, would like to use this product, just don't see any documentation on how to achieve the above goals.

**Comment by francoistanguay** [comment]
We decided to focus on Material as our first design supported as it typically makes more sense in a cross-platform scenario.

Fluent will come next and we're always on the lookout for early adopters.

There might be a way around it but would require a call to discuss further so we can better understand the scope of the app you're trying to build. Feel free to reach out at info @ platform.uno  

Finally, we have somedocs about custom component support here: https://platform.uno/docs/articles/external/figma-docs/learn/developers/custom-components.html
